package.json

```diff
@@ -13,7 +13,10 @@
     "test": "NODE_ENV=development jest",
     "test:git-history": "commitlint --from origin/master --to HEAD",
     "test:lockfile": "node lockFileLint.js",
-    "posttest": "npm run lint && npm run test:git-history && npm run test:lockfile"
+    "posttest": "npm run lint && npm run test:git-history && npm run test:lockfile",
+    "lerna:version": "lerna version",
+    "lerna:publish": "lerna publish from-package --yes",
+    "lerna:deploy":"node lernaDeploy.js"
```

Comment:
There should be a space between `:` and `"node lernaDeploy.js"`

Reply:
Thank you updated a9da5b2